lua/chatcommands/coinflip.lua

```diff
@@ -0,0 +1,89 @@
+CoinFlips = CoinFlips or {}
+RegisterChatCommand({'coin','coinflip'},function(ply,arg)
+	if arg:lower() == "accept" then checkCoinFlipRequest(ply) return end
+	local t = string.Explode(" ",arg)
+	local p = tonumber(table.remove(t))
+	if p == nil then
+		ply:ChatPrint("[orange]!coinflip player points")
+	else
+		local to,c = PlyCount(string.Implode(" ",t))
+		if c == 1 then
+			if ply == to then
+				ply:ChatPrint("[red]You can't coinflip yourself!")
+			elseif p >= 1000 then -- minimum 1000 point coinflip
+				initCointFlip(ply,to,p)
+			else
+				ply:ChatPrint("[red]A coin flip must be a minimum of 1000 points. No pussy bets!")
+			end
+		else
+			ply:ChatPrint("[red]Player "..string.Implode(" ",t).." not found")
+		end
+	end
+end,{global=true,throttle=true})
+
+function initCointFlip(ply,target,amount)
+	if ply:PS_HasPoints(amount) and target:PS_HasPoints(amount) and CoinFlips[ply:SteamID()] == nil then
+		-- Both players have enough points AND an existing coinflip doesn't exist.
+		ply:ChatPrint("[orange]"..to:Nick().." is receiving your coinflip request.")
+		CoinFlips[ply:SteamID()] = {target:SteamID(), amount}
+		timer.Simple(120, CoinFlipRequestTimeout, ply, target) -- 2 minutes for the timeout.
+	elseif CoinFlips[ply:SteamID()] ~= nil then
+		ply:ChatPrint("[red]What the hell r yeh doing bruv? yeh already hae a filp goin!")
+	elseif not ply:PS_HasPoints(amount) then
+		ply:ChatPrint("[red]What the hell r yeh doing bruv? yeh don't hae dooze points!")
+	elseif not target:PS_HasPoints(amount) then
+		-- Should a player be told another player doesn't have enough points?
```

Comment:
No, you can use this to see exactly how much someone has.

Reply:
Any recommendation as to resolve this? Should it just act as if the player never accepted and time out accordingly?